test/drbgtest.c

```diff
@@ -682,6 +684,7 @@ static int test_rand_drbg_reseed(void)
     RAND_DRBG *master, *public, *private;
     unsigned char rand_add_buf[256];
     int rv=0;
+    time_t t;
```

Comment:
The name `before_reseed` instead of `t` would be more self-explaining.

Reply:
Okay.